### PR TITLE
Add Hairflower and Explorer's Labcoat to Loadouts

### PR DESCRIPTION
**Pull request by AeraAuling**
# Description

Very simple PR. This adds the hairflower and Explorer's labcoat to the head loadout and Epistemics job loadouts respectively. This will go to upstream eventually, I've just got way too much open on that side and don't want to accidentally push kitsune stuff with this, so it gets to come here first

---

# TODO

- [x] add hairflower to public head loadouts
- [x] add explorer's labcoat to epi

---

<details><summary><h1>Media</h1></summary>
<p>

Example Media Embed

</p>
</details>

---

# Changelog

:cl:
- add: Epistemics has been given a supply of new labcoats, and everyone has been authorized to wear hairflowers.

### CODE REVIEW

File: Resources/Prototypes/Loadouts/head.yml [hunk 1]
```diff
@@ -70,6 +70,14 @@
   exclusive: true
   items:
     - ClothingHeadHatBellhop
+
+- type: loadout
+  id: LoadoutHeadFlower
+  category: Head
+  cost: 2
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/head.yml:77]: Does it have to cost "2" points just for a flower?
1 seem fine.

File: Resources/Prototypes/Loadouts/head.yml [hunk 2]
```diff
@@ -70,6 +70,14 @@
   exclusive: true
   items:
     - ClothingHeadHatBellhop
+
+- type: loadout
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/head.yml:74]:
```suggestion
- type: loadout # FloofStation
```

File: Resources/Prototypes/Loadouts/Jobs/science.yml [hunk 3]
```diff
@@ -74,6 +74,20 @@
   items:
     - ClothingOuterCoatLabSeniorResearcher
 
+- type: loadout
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Loadouts/Jobs/science.yml:77]:
```suggestion
- type: loadout # Floofstation
```

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Simple first good .yml changes PR, make sure to mark so we dont override it with upsteam merges.

File: Resources/Prototypes/Loadouts/head.yml (hunk 1 above)
Comment (AeraAuling) [reply Resources/Prototypes/Loadouts/head.yml]: lowered the cost to 1 supposing I did this right, should be good

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Fansana — APPROVED** [review]

**Pull request merged**